app/src/main/java/com/aws/amazonlocation/ui/main/explore/ExploreFragment.kt

```diff
@@ -982,8 +999,13 @@ class ExploreFragment :
                                 mViewModel.mCarData?.legs?.let { legs ->
                                     if (!mBottomSheetHelper.isDirectionSearchSheetVisible()) {
                                         legs[0].distance?.let { distance ->
```

Comment:
Should we first check that 0 element is exist?

Reply:
Resolved requested PR review changes.